packages/nvidia_nat_core/src/nat/front_ends/fastapi/response_helpers.py

```
@@ -35,18 +37,21 @@ async def generate_streaming_response_as_str(payload: typing.Any,
 
-        if (isinstance(item, ResponseSerializable)):
-            yield item.get_stream_data()
-        else:
-            raise ValueError("Unexpected item type in stream. Expected ChatResponseSerializable, got: " +
-                             str(type(item)))
+            if (isinstance(item, ResponseSerializable)):
+                yield item.get_stream_data()
+            else:
+                raise ValueError("Unexpected item type in stream. Expected ChatResponseSerializable, got: " +
+                                 str(type(item)))
+    except Exception as e:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)